pages/common/tlmgr-option.md

```diff
@@ -0,0 +1,28 @@
+# tlmgr option
+
+> TeX Live settings manager.
+> More information: <https://www.tug.org/texlive/tlmgr.html>.
+
+- List all TeX Live settings:
```

Comment:
```suggestion
- List all available TeX Live settings:
```

Reply:
I suppose option would be better, however its like describing the color red as the color red, it doesn't add anything + settings is more common

Reply:
Well, setting doesn't add anything either, so on that basis it doesn't matter either way.

Reply:
Still, setting is more common and explicit